Contrib/SA_Score/sascorer.py

```diff
@@ -46,14 +46,15 @@ def numBridgeheadsAndSpiro(mol, ri=None):
   nBridgehead = rdMolDescriptors.CalcNumBridgeheadAtoms(mol)
   return nBridgehead, nSpiro
 
+# 4294967295 was the default length of the Morgan fingerprint
+mfpgen = rdFingerprintGenerator.GetMorganGenerator(radius=2, fpSize=4294967295)
```

Comment:
You should be using `GetSparseCountFingerprint()` below, in which case you do not need to specify the size for the FP here:
```suggestion
mfpgen = rdFingerprintGenerator.GetMorganGenerator(radius=2)
```